goon/browserassets/css/browserOutput.css

```diff
@@ -291,6 +291,8 @@ h1.alert, h2.alert			{color: #000000;}
 .alien					{color: #543354;}
 .noticealien				{color: #00c000;}
 .alertalien				{color: #00c000; font-weight: bold;}
+.terrorspider				{color: #543354;}
```

Comment:
I did use .alien in the past, but someone (a coder or maintainer) requested I change it, so I did. I liked the color, though, so I used the same color.